app/build.gradle

```diff
@@ -5,13 +5,13 @@ plugins {
 }
 
 android {
-    def AXE_DEVTOOLS_APIKEY = axe_apikey
+    def AXE_DEVTOOLS_APIKEY = System.getenv("AXE_MOBILE_API_KEY")
 
     compileSdk 34
 
     defaultConfig {
         applicationId "com.deque.mobile.axedevtoolssampleapp"
-        minSdk 28
+        minSdk 33
```

Comment:
I would recommend keeping it 28 to make sure we are backwards compatible enough.

Reply:
will revert